### PR TITLE
Refactored replacing clipboard contents into a util function

### PR DESCRIPTION
**Pull request by Olian04**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions 
   * Since no testing environment has be established, establishing one in this pr would be out of scope.

### Description of the Change

As requested in my [previous pr](https://github.com/cliffordfajardo/cato/pull/9), I've extracted the clipboard refactoring into its own pr.
The refactoring adds a function called `copyToClipboard` in util.js on the utils object. I then tried to find all occurrences of clipboard manipulation in the code and changed them to use `copyToClipboard` instead.

### Alternate Designs

We could have opted to use a known and tested clipboard library such as [clipboard.js](https://clipboardjs.com/).

### Why Should This Be In Core?

This pr eliminates some code duplication and centralizes util functions to the util.js file.

### Benefits

This pr eliminates some code duplication and centralizes util functions to the util.js file.

### Possible Drawbacks

Tighter coupling between plugins and core application, since some plugins now will use functions from the core libraries utils.

### Applicable Issues

Old pr: https://github.com/cliffordfajardo/cato/pull/9

### CODE REVIEW

File: app/util.js [hunk 1]
```diff
@@ -212,6 +209,14 @@ utils.useAvalableExtensionIcon = function useAvalableExtensionIcon(extension) {
   return icon.url
 }
 
-
+utils.copyToClipboard = function copyToClipboard(value, cb) {
+  document.addEventListener('copy', (event) => {
+      event.preventDefault(); // Prevents the default behavior of copying, ex: pressing Ctrl+C
+      // If we didn't prevent the prevent default, the clipboard would be filled with what ever the user had highlighted on the page.
+      event.clipboardData.setData('text/plain', value);
+      cb(event);
```
Comment (cliffordfajardo) [inline app/util.js:217]: We  should only call `cb` if we have a function. Currently, if we don't pass one and we attempt to only copy data we will get an exception.

```
if(cb) {
  cb(event) 
}
```

**Comment by cliffordfajardo** [comment]
@Olian04 - any updates? If there's no response within the next 7 days, I'm going make a new branch and use your code as a starting point for the completed feature.

**Comment by Olian04** [comment]
@cliffordfajardo - what kind of updates would you expect? The feature is implemented, and I've received no feedback on it.

**Comment by cliffordfajardo** [comment]
@Olian04 - didn't realize github had the comments I wrote in a `pending` state. Comments should be visible now I hope.

**Pull request merged**